storybook/src/components/Switch/Switch.docs.mdx

```diff
@@ -9,7 +9,7 @@ import README from "../../../../packages/css/src/components/switch/README.md?raw
 <Markdown>{README}</Markdown>
 
 It can be used as a standalone component or as a form control element.
-When used as a standalone component provide an `aria-label` or `aria-labelledby` prop to keep it accessible.
+When used as a standalone component provide an `aria-labelledby` property or a descriptive label in an `ams-visually-hidden` element to keep it accessible.
```

Comment:
This is kind of confusing, normally the "aria-labelledby" with refer to an ID of something, but the label of the input is unable to receive an id. So labelledby something other then the label??

Reply:
I'm not sure I follow. Can you make a suggestion?

Reply:
This involves the Switch component and I confused by the use of "label" which is in there but does not describe anything.
```jsx
<div className={clsx('ams-switch', className)}>
        <input {...restProps} className="ams-switch__input" id={inputId} ref={ref} type="checkbox" role="switch" />
        <label className="ams-switch__label" htmlFor={inputId}></label>
      </div>
```

Reply:
So I guess another Form Label component you can use to describe the switch is fine.

Reply:
Multiple `for` attributes seems wrong though.

```jsx
<Field>
<Label htmlFor="switch-with-label">
    Label
  </Label>
  <Switch
    id="switch-with-label"
  />
</Field>
```

Where the switch has an empty label with the same htmlFor attribute.

Reply:
Should it be?

```jsx
<Field>
<Label id="switch-label">
    Label
  </Label>
  <Switch
    aria-labelledby="switch-label"
  />
</Field>
```

Reply:
Maybe we should revert the changes to the Switch docs and focus on Pagination for this PR. That's a lot of work already.

Reply:
Made a ticket for [(aria-)label in Switch](https://gemeente-amsterdam.atlassian.net/browse/DES-924)